playground/backend-integration/__tests__/backend-integration.spec.ts

```diff
@@ -37,7 +37,6 @@ describe.runIf(isBuild)('build', () => {
     const cssAssetEntry = manifest['global.css']
     const scssAssetEntry = manifest['nested/blue.scss']
     const imgAssetEntry = manifest['../images/logo.png']
-    const dirFooAssetEntry = manifest['../../dir/foo.css'] // '\\' should not be used even on windows
```

Comment:
Do you have a reason to remove this test? I think we shouldn't remove this one.

Reply:
In MacOS, `dirFooAssetEntry` is `undefined`, and the case can not pass.

Reply:
Because the behaviour has broken and used the chunk name as the name of the assets, so should test `manifest['bar.css']` and `manifest['foo.css']`. The case has not actually removed.

Reply:
Isn't that because you changed this line?
https://github.com/vitejs/vite/pull/11578/files#diff-4fdf9a690fa55c7c730e03a900f729ce1afb886d64b4280cd4ce0f32d5844074L22-R22
I guess it would work if you revert that change.
I think you can append another entry point for the new test.

Reply:
Even if I revert the change, the chunk name has been used as the CSS asset name, instead of the path before, because this is an entry.